libs/api/domains/cms/src/lib/models/connectedComponent.model.ts

```diff
@@ -0,0 +1,69 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql'
+import graphqlTypeJson from 'graphql-type-json'
+import {
+  ISliceConnectedComponent,
+  ISliceConnectedComponentFields,
+} from '../generated/contentfulTypes'
+
+@ObjectType()
+export class ConnectedComponent {
+  @Field()
+  typename: string
+
+  @Field(() => ID)
+  id: string
+
+  @Field()
+  title: string
+
+  @Field({ nullable: true })
+  type?: string
+
+  @Field(() => graphqlTypeJson, { nullable: true })
+  json?: Record<string, any>
```

Comment:
I believe we intended this json to be temporary while we test out the display of the component, maybe we should indicate that with a comment?

Reply:
Ok, I have added a comment mentioning that this is a temporary field for trying out the first component.